design/overview.md

```diff
@@ -43,7 +43,11 @@ Key concepts that we mention below will be defined here:
 - `Confirmed` - Many actions in Peggy require RLP encoded signatures to be submitted by the `Orchestrators`, an operation is `Confirmed` when it is possible to execute it on Ethereum by collecting and submitting these RLP encoded signatures representing 66% of all voting power encoded in the `Last Valset`.
 - `Observed` - events on Ethereum are considered `Observed` when 66% of the active Cosmos validator set during a given block has submitted an oracle message attesting to seeing the event. Note this is distinct from `Confirmed`! At any given time the set of who can `Observe` an event and who can `Confirm` and event will be slightly different. Since `Confirmed` is based off of the `Last ValSet` and `Observed` is always up to date with the latest validator set.
 - `Validator set delta` - This is a term for the difference between the validator set currently in the Peggy Ethereum contract and the actual validator set on the Cosmos chain. Since the validator set may change every single block there is essentially guaranteed to be some nonzero `Validator set delta` at any given time.
-- `Allowed validator set delta` - This is the maximum allowed `Validator set delta` this parameter is used to determine if a Peggy contract should be adopted and if a validator set update should be sent.
+- `Allowed validator set delta` - This is the maximum allowed `Validator set delta` this parameter is used to determine if the Peggy contract in MsgProposePeggyContract has a validator set 'close enough' to accept. It is also used to determine when validator set updates need to be sent. This is decided by a governance vote _before_ MsgProposePeggyContract can be sent.
+- `Peggy ID` - This is a random 32 byte value required to be included in all Peggy signatures for a
```

Comment:
:+1: This ID is then stored in the contract on ethereum, right?

Reply:
yup as part of the constructor of the contract.